packages/by-name/microsoft/genpolicy/package.nix

```diff
@@ -57,8 +57,12 @@ rustPlatform.buildRustPackage rec {
       ./0006-genpolicy-support-HostToContainer-mount-propagation.patch
       # This patch is a port of https://github.com/kata-containers/kata-containers/pull/10136/files
       # to Microsofts genpolicy.
-      # TODO(miampf): remove when picked up by microsoft/kata-containers fork.
+      # TODO(miampf):
```

Comment:
why change this?

Reply:
It was by accident, when copy-pasting. thanks for pointing out!